src/OmnichannelChatSDK.ts

```diff
@@ -975,8 +976,15 @@ class OmnichannelChatSDK {
                 this.authSettings = authSettings;
             }
 
-            const {PreChatSurvey: preChatSurvey, msdyn_prechatenabled, msdyn_callingoptions} = liveWSAndLiveChatEngJoin;
+            const {PreChatSurvey: preChatSurvey, msdyn_prechatenabled, msdyn_callingoptions, msdyn_conversationmode} = liveWSAndLiveChatEngJoin;
             const isPreChatEnabled = msdyn_prechatenabled === true || msdyn_prechatenabled == "true";
+            
+            if(typeof(liveWSAndLiveChatEngJoin.msdyn_conversationmode) === 'string') {
+                if (liveWSAndLiveChatEngJoin.msdyn_conversationmode.toString() === ConversationMode.PersistentChat.toString()) {
```

Comment:
You can reuse `msdyn_conversationmode` instead of `liveWSAndLiveChatEngJoin.msdyn_conversationmode` since you already declared the values in a previous line

<pre>
const {PreChatSurvey: preChatSurvey, msdyn_prechatenabled, msdyn_callingoptions, <b>msdyn_conversationmode</b>} = liveWSAndLiveChatEngJoin;
</pre>